qml/keys/KeyPad.qml

```diff
@@ -75,6 +75,7 @@ Item {
 
         onItemSelected: {
             UI.keyboardSizeChoice = modelData;
+            maliit_input_method.keyboardSize = modelData;
```

Comment:
This shouldn't be needed if UI.keyboardSizeChoice is binded to maliit_input_method.keyboardSize